### ISSUE TITLE
Metameta fails with example file

### ISSUE THREAD

**Issue by ghost** [pirovc/metameta#14]
Hi,

I tested metameta now with an example file, and it also fails at the metameta step:

* run.sh
```
metameta --configfile test.yaml --use-conda --keep-going --cores 5
```

*) test.yaml
```
workdir: "/home/stephan/work/10_meta_meta/output/"
dbdir: "/home/stephan/work/01_meta_meta_db/"

samples:
  "sample1":
    fq1: "/home/stephan/work/miniconda/data/opt/metameta/sampledata/files/reads/sample_data_archaea_bacteria.1.fq.gz"


databases:
  - archaea_bacteria_201503
tools:
    "clark": "p"
    "kaiju": "p"

threads: 30
gzipped: 1
keepfiles: 1
```

* metametamerge.log
```
- - - - - - - - - - - - - - - - - - - - -
           MetaMetaMerge 1.1
- - - - - - - - - - - - - - - - - - - - -
Input files:
 clark (p) sample1/profiles/archaea_bacteria_201503/clark.profile.out /home/stephan/work/01_meta_meta_db/archaea_bacteria_201503/clark.dbprofile.out
 kaiju (p) sample1/profiles/archaea_bacteria_201503/kaiju.profile.out /home/stephan/work/01_meta_meta_db/archaea_bacteria_201503/kaiju.dbprofile.out
Taxonomy:
 /home/stephan/work/01_meta_meta_db/taxonomy/names.dmp, /home/stephan/work/01_meta_meta_db/taxonomy/nodes.dmp, /home/stephan/work/01_meta_meta_db/taxonomy/merged.dmp
Bins: 4
Cutoff: 0.0001
Mode: linear
Ranks: species
Output file (type): sample1/metametamerge/archaea_bacteria_201503/final.metametamerge.profile.out (bioboxes)
Verbose: False
Detailed: False
- - - - - - - - - - - - - - - - - - - - -

Parsing taxonomy (names, nodes, merged) ...

Reading database profiles ...
 - /home/stephan/work/01_meta_meta_db/archaea_bacteria_201503/clark.dbprofile.out (tsv)
        species - 1461 entries (0 ignored)
        2 taxons with merged entries [94694,1905730]
        Total - 1459 taxons
 - /home/stephan/work/01_meta_meta_db/archaea_bacteria_201503/kaiju.dbprofile.out (tsv)
        species - 2419 entries (0 ignored)
        4 taxons with merged entries [1405,94694,1334193,1905730]
        Total - 2415 taxons

Reading profiles ...
 - sample1/profiles/archaea_bacteria_201503/clark.profile.out (tsv)
        species - 0 entries (0 ignored)
        (WARNING) no valid entries found [species]
Traceback (most recent call last):
  File "/mnt/vdb1/stephan/10_meta_meta/output/.snakemake/conda/266fca67/bin/MetaMetaMerge.py", line 338, in <module>
    main()
  File "/mnt/vdb1/stephan/10_meta_meta/output/.snakemake/conda/266fca67/bin/MetaMetaMerge.py", line 131, in main
    tool = Tools(input_file, identifiers[idx], methods[idx], parse_files(input_file, methods[idx], all_names_scientific, all_names_other, nodes, merged, ranks, args.verbose), ranks, args.verbose)
  File "/mnt/vdb1/stephan/10_meta_meta/output/.snakemake/conda/266fca67/lib/python3.5/site-packages/metametamerge/Tools.py", line 11, in __init__
    Profile.__init__(self, profile, ranks)
  File "/mnt/vdb1/stephan/10_meta_meta/output/.snakemake/conda/266fca67/lib/python3.5/site-packages/metametamerge/Profile.py", line 15, in __init__
    self.profilerank[rankid] = ProfileRank(profile[np.ix_(profile[:,1]==rankid, [0,2,3])],rankid,sum(profile[:,1]==rankid))
IndexError: too many indices for array


```

Thanks for your help in advance,
Stephan

**Comment by ghost** [comment]
I think I found the error - it's probably misuse of binning and profiling
```
tools:
    "clark": "p"  <-- here
    "kaiju": "p"
```

Best,
Stephan

**Comment by pirovc** [comment]
Hi Stephan,

Very probably that's the cause, because the sample data should produce meaningful results. Keep it in mind that if some tool does not have results, meaning the reads didn't match anywhere in the db, such error will still occur. That will be fixed on the v1.2.2.

Best.
Vitor

**Comment by ghost** [comment]
Hi Vitor,

just wanted to let you know that the run completed.

Thanks and looking forward to the updates,
Stephan
